app/models/order.rb

```diff
@@ -18,9 +18,15 @@ def cancel_order
     item_orders.each do |order|
       if order.status == 'fulfilled'
        order.item.inventory += order.quantity
-     end   #we need to test to make sure this is this doing the thing 
+     end   #we need to test to make sure this is this doing the thing
        order.status = 'unfulfilled'
      end
   end
 
+  def order_fulfilled
```

Comment:
Could also call this fulfill_order as it feels more command-y instead of question-y